frontend/src/styles.scss

```diff
@@ -1,6 +1,6 @@
 $fa-font-path: '../fonts';
 
-@import 'font-awesome/scss/font-awesome';
+@import 'font-awesome/css/font-awesome';
```

Comment:
you should be able to use `$fa-font-path` like I did in propack (and probably need to remove styles from tests)

Reply:
Yes, but here the tests are more important, and I like being able to see the components in the browser when debugging them.

Reply:
Oh yes, indeed.